File: algorithms/bubble_sort/code/bubble_sort.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr
# --- IGNORE ---
if __name__ == "__main__":
    arr = []
    number = int(input("Enter number of elements: "))
    for i in range(number):
        element = int(input(f"Enter element {i+1}: "))
        arr.append(element)
    print(bubble_sort(arr))

File: algorithms/bubble_sort/code/bubble_sort.js
function bubbleSort(arr) {
    for (let i = 0; i < arr.length; i++) {
        for (let j = 0; j < arr.length - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                let temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
    return arr;
}

// Example usage:
let numbers = parseInt (prompt("Enter numbers separated by commas:"));
numbers = numbers.split(',').map(Number);

console.log("Unsorted array:", numbers);

let sortedArray = bubbleSort(numbers);
console.log("Sorted array:", sortedArray);
